Exiled.API/Features/Player.cs

```diff
@@ -1072,6 +1072,24 @@ public void DropItem(Inventory.SyncItemInfo item)
             Inventory.items.Remove(item);
         }
 
+        /// <summary>
+        /// Returns true if item is found otherwise returns false.
+        /// </summary>
+        /// <param name="targetItem">The item to search for.</param>
+        /// <returns>Returns true or false.</returns>
```

Comment:
```suggestion
        /// Indicates whether or not the player has an item.
        /// </summary>
        /// <param name="targetItem">The item to search for.</param>
        /// <returns>true, if the player has it; otherwise, false</returns>
```